packages/state/src/index.ts

```diff
@@ -825,6 +827,15 @@ export function buildTemplate(context: GroupMachineContextValue) {
 
           return formatUnit(item.currentValue);
         } else if (item.currentValue.type === "percent") {
+          if (
+            !hasSeenFillPanel &&
+            (item.max === "1fr" ||
+              (item.max.type === "percent" && item.max.value.eq(100)))
+          ) {
+            hasSeenFillPanel = true;
+            return `minmax(${min}, 1fr)`;
+          }
+
```

Comment:
The second big change in this PR is the concept of a "fill" panel. 

During coding this PR I found a problem:

1. render at a small screen size with 1 panels that is min:200px max: 400px
2. resize to much bigger
3. The two panels both get ~50%, at bigger sizes one panel is capped at 400px and the group doesn't fill the space

The "fill" panel fixes this. I found that if we just assign 1 panel to `1fr` and all siblings are pixels/percentages it lays out correctly. When the container grows this fill panel grows to fit the space. 

This is really needed cause we don't update any layout on resize and leave that all to `grid`.

Reply:
In the future we could make the fill panel configurable